code/modules/persistence/storage/smartfridge.dm

```diff
@@ -51,14 +51,7 @@
 
 		// Delete some stacks if we want
 		if(stacks_go_missing)
-			var/fuzzy = rand(-5,5)
```

Comment:
It would be nice if it still had a tiny bit of randomness so it's not super obvious number-wise ingame. Like having 100 of everything turning into 80 of everything seems visually unappealing.

Reply:
I agree with that.
I just wanted to make it so you didn't have to meta-game the correct amount put into it.
Having it be random would make it look a lot better, especially because otherwise everything that hit the cap.
Might end up with exactly the same amount. I'll see if I can find a way to work that into the new formula.